cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaSetOperator.java

```diff
@@ -49,4 +58,64 @@ public static boolean needsRollingUpdate(StatefulSetDiff diff) {
         }
         return false;
     }
+
+    private void revertVolumeChanges(StatefulSet current, StatefulSet desired) {
+
+        Container currentKafka =
+                current.getSpec().getTemplate().getSpec().getContainers().stream().filter(c -> c.getName().equals("kafka")).findFirst().get();
+        Container desiredKafka =
+                desired.getSpec().getTemplate().getSpec().getContainers().stream().filter(c -> c.getName().equals("kafka")).findFirst().get();
+
+        desiredKafka.setVolumeMounts(currentKafka.getVolumeMounts());
+
+        StatefulSetBuilder ssBuilder = new StatefulSetBuilder(desired);
```

Comment:
Why declare `ssBuilder` at all?

Reply:
Yeah it's useless ... I was trying to understand in different ways how to edit env var and I forgot to delete it.